comps/animation/animation.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import pathlib
+import platform
+import subprocess
+import time
+
+import cv2
+import ffmpeg
+import numpy as np
+
+# Wav2Lip-GFPGAN
+import requests
+import Wav2Lip.audio as audio
+import Wav2Lip.face_detection as face_detection
+from basicsr.utils import imwrite
+from gfpgan import GFPGANer
+from Wav2Lip.models import Wav2Lip
+
+cur_path = pathlib.Path(__file__).parent.resolve()
+comps_path = os.path.join(cur_path, "../")
+
+# Habana
+import habana_frameworks.torch.core as htcore
```

Comment:
what if we want to run on Xeon without habana installation?

Reply:
We will specify an argument "device" to pick "hpu", "cpu", or "cuda" in `animation.py`
![image](https://github.com/user-attachments/assets/5e78ba60-4685-4c35-b847-af560a9e0e0e)